app/uk/gov/hmrc/gform/formtemplate/Rewriter.scala

```diff
@@ -56,20 +57,31 @@ trait Rewriter {
     }
 
   private def validateAndRewriteIncludeIf(formTemplate: FormTemplate): Either[UnexpectedState, FormTemplate] = {
```

Comment:
This seems to rewrite validIfs too. Shouldnt the name reflect that? `validateAndRewriteValidIdsAndIncludeIfs`

Reply:
Changed to more precise `validateAndRewriteBooleanExprs`